### PR TITLE
change files UWU

### PR DESCRIPTION
**Pull request by ranidu-harshana**

### CODE REVIEW

File: cc/UWU/2022-current.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# UWU - Uva Wellassa University
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]: If you have a logo you can add it as well.

File: cc/UWU/2022-current.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+Mozilla Campus Club - UWU is a community organization which supports university students to contribute Free and Open Source Software (FOSS) projects to make global web better.
+The world’s communications infrastructure needs to contend with pervasive surveillance, censorship, and misinformation. The need for us, as foundations, to join in advancing the university students in this rapidly evolving digital environment is greater than ever.
+The declaration We are seeking like a Mozilla Campus club-UWU to make the internet an open, secure, and fair space protecting free expression and privacy, and advancing economic opportunity, knowledge exchange, and civic engagement that everyone can access and afford.
+We look forward to working with established partners and new ones with a common ambition to curtail the use of digital technologies as tools for repression and manipulation, and to strengthen their use in building more just, accessible, and open societies.
+
+
+
+## Members
+
+* Amal Vishwajith
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]: The lead name is supposed to be bold. (as on the template)

File: cc/UWU/2022-current.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ - Sunera Sunilaka
+ - Vickneshwarasarma Yogeswaraiyer
+ - Hashini Munasinghe
+ - Chethana Wickramaarachchi
+ - Rajitha Rathnayaka
+
+## How to join?
+
+- Go to [Mozilla Campus Club - UWU](https://mozillauwu.github.io/) official website
+- On top of the page click on the "join club" button. that will navigate you to the membership section
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]:
```suggestion
- On top of the page click on the `join club` button. That will navigate you to the membership section
```

File: cc/UWU/2022-current.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- Click on the [Register Now](https://mozillauwu.page.link/reg) button 
+- Now fill the form and submit. thats all!
+
+
+## Links
+
+- [https://mozillauwu.github.io/](https://mozillauwu.github.io/)
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]:
```suggestion
- [Website](https://mozillauwu.github.io/)
```

**Review by dhanushkac — CHANGES_REQUESTED** [review]

File: cc/UWU/2022-current.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
+# UWU - Uva Wellassa University ![Final_Logo](https://user-images.githubusercontent.com/68012743/202668062-e0190d81-9b80-4c03-8692-4241d2a1f41a.png)
+
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]:
```suggestion
<img src="https://user-images.githubusercontent.com/68012743/202668062-e0190d81-9b80-4c03-8692-4241d2a1f41a.png" width="20%" />
```

This image is too large. Adjust width accordingly.

File: cc/UWU/2022-current.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+Mozilla Campus Club - UWU is a community organization which supports university students to contribute Free and Open Source Software (FOSS) projects to make global web better.
+The world’s communications infrastructure needs to contend with pervasive surveillance, censorship, and misinformation. The need for us, as foundations, to join in advancing the university students in this rapidly evolving digital environment is greater than ever.
+The declaration We are seeking like a Mozilla Campus club-UWU to make the internet an open, secure, and fair space protecting free expression and privacy, and advancing economic opportunity, knowledge exchange, and civic engagement that everyone can access and afford.
+We look forward to working with established partners and new ones with a common ambition to curtail the use of digital technologies as tools for repression and manipulation, and to strengthen their use in building more just, accessible, and open societies.
+
+
+
+## Members
+
+* **Amal Vishwajith**
```
Comment (dhanushkac) [inline cc/UWU/2022-current.md]:
```suggestion
- **Amal Vishwajith**
```

File: cc/UWU/2022-current.md (hunk 5 above)
Comment (dhanushkac) [reply cc/UWU/2022-current.md]: @ranidu-harshana why was this marked resolved?

Comment (ranidu-harshana) [reply cc/UWU/2022-current.md]: Sorry for the mistake aiya

**Review by dhanushkac — APPROVED** [review]
LGTM 👍

**Pull request merged**